api/app/health.py

```diff
@@ -0,0 +1,44 @@
+""" Module to perform health checks.
+"""
+import os
+import json
+import logging
+import logging.config
+import requests
+from app import config
+
+LOGGING_CONFIG = os.path.join(os.path.dirname(__file__), 'logging.json')
```

Comment:
Please remove the logging config section - it's configured in main.py - you only need the logger.